include/nbl/builtin/hlsl/workgroup/broadcast.hlsl

```diff
@@ -22,7 +22,7 @@ namespace workgroup
  * and then all invocations access that index.
  */
 template<typename T, class SharedAccessor>
-T Broadcast(in T val, NBL_REF_ARG(SharedAccessor) accessor, in uint id)
+T Broadcast(NBL_CONST_REF_ARG(T) val, NBL_REF_ARG(SharedAccessor) accessor, NBL_CONST_REF_ARG(uint) id)
```

Comment:
btw, sub 32-bit types are not worth declaring as const references, just use `const T`